core/iwasm/aot/aot_runtime.c

```diff
@@ -4104,6 +4105,48 @@ aot_frame_update_profile_info(WASMExecEnv *exec_env, bool alloc_frame)
 #endif /* end of WASM_ENABLE_AOT_STACK_FRAME != 0 */
 
 #if WASM_ENABLE_DUMP_CALL_STACK != 0
+void 
+aot_iterate_callstack(WASMExecEnv *exec_env, const wasm_frame_callback frame_handler, void* user_data)
+{
+/*
+* Note for devs: please refrain from such modifications inside of aot_iterate_callstack
+ * - any allocations/freeing memory
+ * - dereferencing any pointers other than: exec_env, exec_env->module_inst,
+ * exec_env->module_inst->module, pointers between stack's bottom and top_boundary
+ * For more details check wasm_iterate_callstack in wasm_export.h
+*/
+    if (!is_tiny_frame(exec_env)) {
+        //TODO: support standard frames
```

Comment:
should we print an error/warning here? Or ideally, can we provide an implementation of that? I think there's already a code in this file to iterate through both types of frames so can be used as a reference

Reply:
I'll take a look if I can implement it as well

Reply:
I added support for standard frames, the same way as it's done for wasm